### PR TITLE
added AOSP - Issue tracker

### PR DESCRIPTION
**Pull request by shanraisshan**
added AOSP - Issue tracker
strikeout depreciated Vulnerability Google Doc
removed Root Exploits (from Drozer issue #56)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,11 +210,12 @@ Tutorials
...
 ----
-1. [Vulnerability Google Doc](https://docs.google.com/spreadsheet/pub?key=0Am5hHW4ATym7dGhFU1A4X2lqbUJtRm1QSWNRc3E0UlE&single=true&gid=0&output=html)
-* [Root Exploits (from Drozer issue #56)](https://github.com/mwrlabs/drozer/issues/56)
-* [Android Security Bulletins](https://source.android.com/security/bulletin/)
+
+1. [Android Security Bulletins](https://source.android.com/security/bulletin/)
 * [Android's reported security vulnerabilities](https://www.cvedetails.com/vulnerability-list/vendor_id-1224/product_id-19997/Google-Android.html)
 * [Android Devices Security Patch Status](https://kb.androidtamer.com/Device_Security_Patch_tracker/)
+* [AOSP - Issue tracker](https://code.google.com/p/android/issues/list?can=2&q=priority=Critical&sort=-opened)
+* ~~[Vulnerability Google Doc](https://docs.google.com/spreadsheet/pub?key=0Am5hHW4ATym7dGhFU1A4X2lqbUJtRm1QSWNRc3E0UlE&single=true&gid=0&output=html)~~
```
Comment (ashishb) [inline README.md]: Why strike it out?
This link is not dead.

Comment (shanraisshan) [inline README.md]: 2013 - you can consider it obsolete may be

Comment (ashishb) [inline README.md]: Yeah but that's true for many other tools as well since none of them are being actively developed.

Comment (shanraisshan) [inline README.md]: Removed strikethrough

**Comment by ashishb** [comment]
This looks pretty good at this point, one last small change and I'll merge it.

**Comment by shanraisshan** [comment]
I think we should keep this category name as generic  **Malware** only, so that we can also share other stuff as well. WDYT?

**Comment by ashishb** [comment]
I don't have a strong preference, so, let's go by your suggestion.

**Pull request merged**